src/ui-kit/nav/footer.tsx

```diff
@@ -1,25 +1,65 @@
-'use client';
-
-import { YMaps, Map, Placemark } from 'react-yandex-maps';
+/* eslint-disable max-len */
+import Image from 'next/image';
+import Link from 'next/link';
 
 export function Footer() {
   return (
-    <footer className="flex h-[500px] mt-20 bg-gray justify-between items-center">
-      <div>
-        Info about mag
-      </div>
-      <div className="m-10">
-        <YMaps>
-          <Map
-            width={400}
-            height={400}
-            defaultState={{ center: [53.890585, 25.300206], zoom: 17 }}
-          >
-            <Placemark geometry={[53.890585, 25.300206]} />
-          </Map>
-        </YMaps>
+    <footer className="max-w-1140 w-full px-15 bg-black">
+      <div className="container mx-auto px-4 pt-20 pb-10">
+        <ul className="text-white font-semibold text-3xl flex justify-between mb-24">
+          <li>
+            <Link href="#">
+              <p className="block mb-6">каталог</p>
+            </Link>
+            <Link href="#">
+              <p className="block mb-6">о нас</p>
```

Comment:
Use [gap](https://developer.mozilla.org/en-US/docs/Web/CSS/gap) css prop instead of mb-6 everywhere